### ISSUE TITLE
Update Project Profile: Civic Tech Jobs

### ISSUE THREAD

**Issue by kcoronel** [hackforla/website#4127]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
The Civic Tech Jobs project page needs to be updated to reflect accurate team information. 

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of the `_projects/civic-tech-jobs.md` file and find the `leadership` variable.
- [x] Remove the following members and their information:
  - [x] Syed Danish Imran
  - [x] Kevin Park
  - [x] Grace Lin
  - [x] Maria Elizabeth Rodriguez Beltran
  - [x] Stephanie G. Stahlberg
  - [x] Helene Dendor
  - [x] Summer Kim
- [x] Add the new team members `name`s, `role`s, `slack` links, `github` links, and `picture` to the `leadership` variable
  - [x] Name: Ava Li
  GitHub Handle: Aveline-art
  Role: Development Lead
  Slack Member ID: U01B73XGCKV
  - [x] Name: Noor Grewal
  GitHub Handle: gjagnoor
  Role: Full Stack Developer
  Slack Member ID: U045Q8PD1B5
  - [x] Name: Jen Chung
  GitHub Handle: jenchuu
  Role: UX/UI Design Lead
  Slack Member ID: U045Q8PD1B5
  - [x] Name: Tin Wei Chung
  GitHub Handle: TCUX
  Role: UX/UI Designer
  Slack Member ID: U03P6Q6FSQ5
  - [x] Name: Lu Feng
  GitHub Handle: fenglugithub
  Role: UX/UI Designer
  Slack Member ID: U03NV47TG4X
  - [x] Name: Gabriel Vicencio
  GitHub Handle: geibu
  Role: UX/UI Designer
  Slack Member ID: U03P33ZNPGW
  - [x] Name: Melinda Sukosd
  GitHub Handle: melkosm
  Role: UX Research Lead
  Slack Member ID: U03T1G9F46P
  - [x] Name: Beth Williford
  GitHub Handle: BethBethBeth
  Role: UX Researcher
  Slack Member ID: U03T1G9F46P
  - [x] Name: Genggeng Zhang
  GitHub Handle: gzhang5240
  Role: UX Researcher
  Slack Member ID: U03UYEVMV37
  - [x] Name: Karen Coronel
  GitHub Handle: kcoronel
  Role: Product Manager
  Slack Member ID: U010KUGJATC
  - [x] Name: Matt Pereira
GitHub Handle: @MattPereira
Role: Developer
Slack Member ID: U045XKKB5DH
  - [x] Name: Bitian Zhang
GitHub Handle: @bzzz-coding
Role: Developer
Slack Member ID: U04H6PYK14N
- [x] Verify the changes by viewing the [Civic Tech Jobs project page](https://www.hackforla.org/projects/civic-tech-jobs) on your local environment and include a screenshot with your pull request

### Instructions
- For the github link, append the team member's github handle to https://github.com/
- For the picture link, append the team member's github handle to https://avatars.githubusercontent.com/
- For the slack link, append the team member's slack ID to https://hackforla.slack.com/team/

### Resources
- [civic-tech-jobs.md](https://github.com/hackforla/website/blob/gh-pages/_projects/civic-tech-jobs.md)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

**Comment by ExperimentsInHonesty** [comment]
@kcoronel @salimays we are confused by this issue, it has no data in it.  Do you want us to delete it, so you can open a new one?

**Comment by kcoronel** [comment]
@ExperimentsInHonesty this is a draft, we are asking our leads to get us this information. Please do not delete.

**Comment by ExperimentsInHonesty** [comment]
@kcoronel Thanks, I added the `waiting on stakeholder` label.  You can add the `ready for product` label when you want us to look at it.  In the meantime, we will be able to ignore it. 😴😄

**Comment by ExperimentsInHonesty** [comment]
@kcoronel and @salimays Please update your meeting time data in by logging into the admin link at VRMS.io (https://vrms.io/login).  You both are permissioned there for that.  Here is the [VRMS user guide](https://github.com/hackforla/VRMS/wiki/User-Guide).  If you encounter any trouble, please reach out to the VRMS product team and make an appointment for with them to walk you through it.

**Comment by ExperimentsInHonesty** [comment]
I have verified the information they want to change is approved.  This is ready for dev leads to make a good small issue out of it.

**Comment by jdingeman** [comment]
Hi there, our project Civic Tech Jobs wants to update our current project team from our [project profile](https://www.hackforla.org/projects/civic-tech-jobs). 

### Please Remove: 
Name: [Syed Danish Imran](https://hackforla.slack.com/team/U020Z27JB1S)
Role: Product Manager
Name: [Kevin Park](https://hackforla.slack.com/team/U02B1MG90PK)
Role: Backend Developer
Name: [Grace Lin](https://hackforla.slack.com/team/U02AE5H5BJ9)
Role: UX/UI Designer
Name: [Maria Elizabeth Rodriguez Beltran](https://hackforla.slack.com/team/U035B853S4W)
Role: UX Researcher
Name: [Stephanie G. Stahlberg](https://hackforla.slack.com/team/U02H06YBR7F)
Role: UX Researcher
Name: [Helene Dendor](https://hackforla.slack.com/team/U02C41H7919)
Role: UX Researcher
Name: [Summer Kim](https://hackforla.slack.com/team/U0374ALAX2Q)
Role: UX Researcher

### Please Update: 

Ava Li
Role: Development Lead

Noor Grewal
Role: Full Stack Developer

Jen Chung
Role: UX/UI Design Lead 

Tin Wei, Chung 
Role: UX/UI Designer

Lu Feng
Role: UX/UI Designer

Gabriel Vicencio
Role: UX/UI Designer

Melinda Sukosd
Role: UX Research Lead 

Beth Williford
Role: UX Researcher

Genggeng Zhang
Role: UX Researcher

Karen Coronel 
Role: Product Manager

**Comment by Aveline-art** [comment]
Hi everyone! Dev Lead for CivicTechJobs here. I would like to work on this ticket personally, just for expediency and I expect slight changes to our...changes. If that is alright, please ping me in slack `@Ava Li` (I no longer receive notifications from this repo due to being subscribed to too many old tickets).

**Comment by ExperimentsInHonesty** [comment]
Hi @Aveline-art If you have changes, your team can submit a new issue once this issue is closed.  I anticipate that the issue will get addressed in 48 hours or less.

The reason why we don't let customers (HfLA projects are our customers) make changes directly, is that would circumvent our approval process that all project changes go through.

Thank you for your understanding.

I am sending this as a comment in your slack now.

**Comment by janice87** [comment]
i. Availability: Mon - Wed 5-9PM EST
ii. ETA: 4/5/23 EOD

**Comment by Aveline-art** [comment]
Is there a way where we can stay this ticket and move it back to it's last status so more information can be added. It takes multiple rounds of back and forth for a ticket to even be made. And it looks like 48 hours is not an accurate estimate? Just looking at the past couple of tickets, it takes 3-4 weeks from opening the ticket to completion? All I really need is to add one more volunteer who spoke to me only mid last week, and will join the team tomorrow. If this ticket was opened just a week later, that would have saved me 3-4 weeks of work. This is frustrating when all I want is to showcase the hardworking volunteers of our team.

**Comment by jdingeman** [comment]
@Aveline-art - Hi Ava, can you please add the volunteer's information as a comment and I will get in touch with the dev.

**Comment by Aveline-art** [comment]
Okay these are the individuals to add:

Name: Matt Pereira
GitHub Handle: @MattPereira  
Role: Developer
Slack Member ID: U045XKKB5DH

Name: Bitian Zhang 
GitHub Handle: @bzzz-coding 
Role: Developer
Slack Member ID: U04H6PYK14N
